webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -500,6 +500,31 @@ public function viewAction(Request $request, int $submitId): Response
                 'after' => 15,
                 'url' => $this->generateUrl('jury_submission', ['submitId' => $submission->getSubmitid()]),
             ];
+        } else {
```

Comment:
Should this not be `elseif $selectJudging != $currentJudging` (pseudo)?

Reply:
Actually, I think it is still useful. But I downgraded the severity a little bit and changed the message:
![image](https://user-images.githubusercontent.com/418721/196248575-2946a099-f2ad-4153-a8db-10a51d34e82a.png)

WDYT?